integrations/docker/images/chip-build-esp32/Dockerfile

```diff
@@ -10,7 +10,7 @@ RUN set -x \
     && : # last line
 
 RUN set -x \
-    && git clone --depth 1 --recursive -b v4.3 https://github.com/espressif/esp-idf.git /tmp/esp-idf \
+    && git clone --depth 1 --recursive -b release/v4.4 https://github.com/espressif/esp-idf.git /tmp/esp-idf \
```

Comment:
https://github.com/espressif/esp-idf/tree/release/v4.4 seems to be a branch.

Please use a tag or fix a SHA for the checkout. The reason for this is that we want our docker image builds to be as stable as possible: a build today should be the same as the build of the same dockerfile 1 month from now.

We cannot guarantee that when the checkouts are a moving target like branches which may contain any checkins.

Reply:
Added fixed commit.